pages/common/2to3.md

```diff
@@ -0,0 +1,28 @@
+# 2to3
+
+> Automated Python 2 to 3 code translation.
+> More information: <https://docs.python.org/3/library/2to3.html>.
+
+- Print changes that would be performed:
+
+`2to3 {{file.py}}`
+
+- Convert Python 2 file to Python 3:
+
+`2to3 -w {{file.py}}`
+
+- Convert specified Python 2 features to Python 3:
+
+`2to3 -w {{file.py}} -f {{raw_input}} -f {{print}}`
+
+- Convert all features except the specified ones to Python 3:
+
+`2to3 -x {{raw_input}} -f {{print}} example.py`
```

Comment:
Should the 2nd argument be `--nofix`?
```suggestion
`2to3 --nofix={{raw_input}} --fix={{print}} example.py`
```

Reply:
@navarroaxel sure.  I just copied from the docs page.